src/sequences.h

```diff
@@ -141,6 +171,41 @@ class Sequences
 };
 
 
+class PhaseProbs
```

Comment:
nit: the indentation of the class is inconsistent. Is it possible to follow the style used in the rest of the code base? That is, indent of 4-spaces?